### PR TITLE
1904 avoid using dofpoints compute them on the fly

### LINKED ISSUE #1904: Avoid using dofPoints, compute them on the fly
we should create a geomap context comptuting the point coordinates (POINT context) and use it  in an element/face loops. 

The issues with dofpoints() are : 
- one call will build all dofpoints
- when the mesh move, the developper should call the rebuild of all dofpoint else we get the node without the moving

see #1879 and replace the use of dofPoints in `opselt()`

### PR DESCRIPTION
**Pull request by prudhomm**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/feelpp/feelpp/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your changes?
- [x] Have you successfully run the Feel++ testsuite with your changes locally?
- [x] Have you written Doxygen comments in your contribution ?

-----
- closes #1904

### CODE REVIEW

**Review by lberti — APPROVED** [review]
It's better to wait for @vincentchabannes's confirmation before merging

**Review by vincentchabannes — APPROVED** [review]

**Pull request merged**
